deck-o-cards.js

```diff
@@ -1,49 +1,38 @@
-// 1. Make the function deck_o_cards assemble an array of cards using the provided suits and values arrays.
-// Each card in the deck should be an object formatted as: {suit: 'hearts', value: 'A'}
 function deck_o_cards() {
   var values = [2, 3, 4, 5, 6, 7, 8, 9, 10, 'Jack', 'Queen', 'King', 'Ace'];
   var suits = ['hearts', 'diamonds', 'clubs', 'spades'];
 
-  var cards = []; // deck
-  var shuffledCards = []; // deck shuffled
+  var cards = [];
+  var shuffledCards = [];
 
-  // Make 52 card objects and store them in the "cards" array
-  // Hint: use 2 for loops
-
-
-  // 2. Shuffle the cards
-  // Hint: shuffle function is already defined below
-
-
-  // Pull the top card from the newly shuffledCards
-
-
-  // 3. Print the results:
-  // "The deck has {n} cards"
-  // "The top card is the {value} of {suit}"
+  for (var i = 0; i < suits.length; i++) {
+    for (var j = 0; j < values.length; j++) {
+      var card = {}; 
```

Comment:
You can both create AND set this object at once.

```js
var card = {
   suit: suits[i],
   value: values[j]
}
```

Reply:
Yeah I legit didnt see the bonus for some reason until I was looking at other peoples codes

I think I uploaded wrong version because I had that card object setting them